src/providers/seventv/SeventvEmotes.hpp

```diff
@@ -75,6 +75,7 @@ class SeventvEmotes final
     std::shared_ptr<const EmoteMap> globalEmotes() const;
     boost::optional<EmotePtr> globalEmote(const EmoteName &name) const;
     void loadGlobalEmotes();
+    void setEmotes(std::shared_ptr<const EmoteMap> emotes);
```

Comment:
I think this should be `setGlobalEmotes` (since the corresponding load is called `loadGlobalEmotes`).

```suggestion
    void setGlobalEmotes(std::shared_ptr<const EmoteMap> emotes);
```